samples/core/external_files/external_files.py

```diff
@@ -0,0 +1,86 @@
+import os
+import kfp
+from kfp import dsl
+
+
+def json_op(json: str):
+    op = dsl.ContainerOp(
```

Comment:
Constructing `dsl.ContainerOp` instance directly in user code is discouraged, and we should avoid adding more tech debts.

https://github.com/kubeflow/pipelines/blob/37da6257a38333d94a373511d90b0e101b84d176/sdk/python/kfp/dsl/_container_op.py#L1108-L1115

Reply:
No, ContainerOp has been deprecated.
https://github.com/kubeflow/pipelines/pull/4166

we are planning to update all existing samples, but haven't finished

Reply:
I see. I'm closing this issue.